src/rpc.server.ts

```diff
@@ -53,6 +58,22 @@ export class RPC implements IUsersServer {
         }
     }
 
+    async getApproverInfo(call: grpc.ServerUnaryCall<GetApproverInfoRequest>, callback: grpc.sendUnaryData<GetApproverInfoResponse>) {
+        await wrapper<GetApproverInfoRequest, GetApproverInfoResponse>(RPC.getApproverInfoHandler, call, callback);
+    }
+
+    static async getApproverInfoHandler(call: grpc.ServerUnaryCall<GetApproverInfoRequest>) {
+        const userID: string = call.request.getId();
+        const info: IApproverInfo = await RPC.approvalClient.getApproverInfo(userID);
+        const reply: GetApproverInfoResponse = new GetApproverInfoResponse();
+        if (!info) {
```

Comment:
how could that occur if you have a good error handling in the getApproverInfo function?

Reply:
it shouldn't happen.

Reply:
removed